scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -139,6 +142,11 @@ public override void OnInspectorGUI()
                 var time = t.InitializationTime.Value;
                 EditorGUILayout.LabelField($"Initialized in: {(int)time.TotalSeconds}.{time.Milliseconds.ToString("D3")}s");
             }
+
+            if (GUI.changed)
+            {
+                EditorUtility.SetDirty(target);
+            }
```

Comment:
I've realized that this will set the scene as dirty whenever the GUI changes, even if the serialized data doesn't change. So it will report that the scene needs to be saved when there are no changes. Any ideas how to detect this in a better way than `GUI.changed`? 

This is still better than previously where we lost changes sometimes.

Reply:
Sorry but I don't know of any better way. Looks like something which should be trivial enough, but very well is not.

Reply:
It appears to only be the case when the foldouts/buttons are clicked. So dropdown's, tooltips don't set `GUI.changed`.

Reply:
The best way I see is to have a copy of the previous options and compare them if `GUI.changed` to check for changes. However, this seems like a bit of a overkill and requires various stuff with cloning and `Equals`. For now I would leave it like above.

Reply:
Implemented in 5bda99031c315006d66ea14aa8c7c949c7dabd6e